public/data/vickygupta.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Vicky Gupta",
+    "type": "personal",
+    "bio": "I am Vicky Gupta, currently pursuing my Engineering majoring in Electronics & telecommunication from Atharva college of engineering Institute. I am a tech & an open-source enthusiast. I always like to interact with people and open to collaborating on projects and innovative & disruptive ideas",
+    "avatar": "https://avatars.githubusercontent.com/u/55152616?v=4",
```

```suggestion
    "avatar": "https://github.com/viki4545.png",
```

Inculde short url for avatar instead of long url.

---

Thankyou for correcting me.

---

@viki4545 I didn't correct you as of such, both links work, it's just the short link looks better :)

---

@KendallDoesCoding  yes, i have not read the instruction below that is why i put this link.